lib/DXIL/DxilModule.cpp

```diff
@@ -116,6 +116,7 @@ DxilModule::DxilModule(Module *pModule)
 {
 
   DXASSERT_NOMSG(m_pModule != nullptr);
+  m_pModule->pfnResetDxilModule = &DxilModule_ResetModule;
```

Comment:
Can we move this line after the RemoveGlobal one to prevent a conflict with the fix in the internal repo and align with the HLModule order?